hazelcast/src/test/java/com/hazelcast/config/HostVerificationConfigTest.java

```diff
@@ -0,0 +1,16 @@
+package com.hazelcast.config;
```

Comment:
All new test classes are missing the copyright header and a newline at the EOF (we don't have CheckStyle for tests yet, but this wouldn't pass).